### ISSUE TITLE
Draft customer journey pipeline in Hubspot

### ISSUE THREAD

**Issue by o-ram** [cal-itp/customer-success#531]
Building off of #496 we now need to start testing and filling in this pipeline.

Rough steps include:
- [ ] Testing (_in progress_)
- [ ] Review/walkthrough with Caitlin 
- [ ] Document

**Comment by laneymangan** [comment]
Account Managers to review [this doc](https://docs.google.com/document/d/1knasWP_XJ7djo_POeAlCDVYLtw-WNPPa5biFPATpNX0/edit), as requested by Caitlin in Slack on 10/4. Caitlin requested that we review this by end of week (10/11) (cc @mariearaneta @ninadinh-la @o-ram)

**Comment by ninadinh-la** [comment]
@laneymangan I reviewed and left my comments on the doc!

**Comment by laneymangan** [comment]
Team met with Caitlin 12/9 to review the proposed [Customer Journey](https://docs.google.com/document/d/1crx_jlUxdDExfEcGVjErfQhKrH0Aqpbs34ivQ45_-xk/edit?tab=t.0#heading=h.f59yd9dv2381)

Action Item: team will meet on 1/8 to discuss Roadmap Objectives

**Comment by AnthonyRollins** [comment]
We are currently building out a report that would showcase more and more the customer journey.

**Comment by caitlinharper** [comment]
#617 
#661 
_Next subitems_

- See Olivia's merged Payments/Benefits pipeline
- Make tweaks to other pipelines as needed
- Pull the report(s) again to ensure we get what we need
- Either build it in Hubspot or keep it in Google Sheets

**Comment by AnthonyRollins** [comment]
@caitlinharper I feel as though we can close out this issue as we pivoted away from using Hubspot for this. Is that right?

**Comment by caitlinharper** [comment]
@AnthonyRollins Yeah let's close it. If anything, the AMs can decide what makes sense for them as far as tags/reports, rather than building an actual pipeline.
